lib/spack/spack/util/environment.py

```diff
@@ -980,18 +980,27 @@ def environment_after_sourcing_files(*files, **kwargs):
             only when the previous command succeeds (default: ``&&``)
     """
     # Set the shell executable that will be used to source files
-    shell_cmd = kwargs.get("shell", "/bin/bash")
-    shell_options = kwargs.get("shell_options", "-c")
-    source_command = kwargs.get("source_command", "source")
-    suppress_output = kwargs.get("suppress_output", "&> /dev/null")
+    if sys.platform == "win32":
+        shell_cmd = kwargs.get("shell", "cmd.exe")
+        shell_options = kwargs.get("shell_options", "")
+        suppress_output = kwargs.get("suppress_output", "")
```

Comment:
@johnwparent This got merged in with install tests. Disregard changes for this file.